src/styles/_header.scss

```
@@ -0,0 +1,276 @@
+.header {
+  position: relative;
+  width: 100%;
+
+  height: clamp(520px, 80vh, 840px);
+  margin: 0 auto;
+  color: #fff;
+  background: url('../images/gallery-main.png') center/cover no-repeat;
+}
+
+.header::after {
+  content: '';
+  position: absolute;
+  inset: 0;
+  background: linear-gradient(
+    180deg,
+    rgba(0, 0, 0, 0.55),
+    rgba(0, 0, 0, 0.35) 40%,
+    rgba(0, 0, 0, 0.55)
+  );
+  pointer-events: none;
+}
+
+.header__inner.container {
+  grid-column: 1 / -1;
+}
+
+.header__inner {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  top: 50px;
+  height: 65px;
+  z-index: 3;
+  width: 100%;
+}
+
+.header__logo {
+  margin: 0;
+}
+
+.header__logo-img {
+  height: 65px;
+  width: auto;
+  object-fit: contain;
+  display: block;
+}
+
+.header__actions {
+  margin-left: auto;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  gap: 55px;
+} 
+
+.header__icon {
+  display: grid;
+  place-items: center;
+  border: none;
+  background: none;
+  color: #fff;
+  cursor: pointer;
+  transition: transform var(--ease);
+  inline-size: 32px;
+  block-size: 32px;
+  line-height: 1;
+}
+
+.header__icon:hover,
+.header__icon:focus-visible {
+  transform: translateY(-1px);
+}
+
+.header__icon--tel {
+  background: no-repeat center/18px
+    url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' fill='white' viewBox='0 0 24 24'><path d='M6.62 10.79a15.464 15.464 0 006.59 6.59l2.2-2.2a1 1 0 011.01-.24c1.12.37 2.33.57 3.58.57a1 1 0 011 1V20a1 1 0 01-1 1C10.07 21 3 13.93 3 5a1 1 0 011-1h3.5a1 1 0 011 1c0 1.25.2 2.46.57 3.59a1 1 0 01-.24 1.01l-2.21 2.19z'/></svg>");
+}
+
+.header__icon--menu {
+  font-size: 20px;
+  line-height: 1;
+}
+
+@media (max-width: 743.98px) {
+  .header__icon {
+    inline-size: 24px;
+    block-size: 24px;
+  }
+
+  .header__icon--tel {
+    background-size: 14px;
+  }
+
+  .header__icon--menu {
+    font-size: 16px;
+  }
+}
+
+.header__title {
+  position: absolute;
+  inset: 0;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  padding-top: var(--hdr-title-pt);
+  text-align: center;
+  z-index: 2;
+
+  @media (max-width: 743.98px) {
+    --hdr-title-pt: 200px;
+  }
+
+  @media (min-width: 744px) and (max-width: 1199.98px) {
```

strange max size + we don't need it 
we can either use min-width and scale up or max-width and scale down

```suggestion
  @media (min-width: 744px) {
```